_pages/ato/background.md

```diff
@@ -1,16 +1,45 @@
 
-* Governed by the [Federal Information Security Management Act of 2002](http://csrc.nist.gov/drivers/documents/FISMA-final.pdf) (FISMA)
-    * FISMA does a few things. It puts the Chief Information Security Officer in charge of consulting the agency head with the process they should have for the security clearance process. FISMA also empowers the agency head with accepting all the risk posed by information systems.
-        * Good news is that agency head can delegate that authority at their choosing.
-    * This agency-specific process then binds agency heads with the process that they must then follow.
-        * In practice, what this means is setting up a series of controls for each new system that will be launched.
-* 18F simplifies this process by implementing the bulk of the controls [at the infrastructure level](../../infrastructure/aws/) to the AWS account. This is reflected in a baseline minimum controls we’ve created.
-    * Controls can range from human controls to business processes to mechanical ones.
+A key role in the RMF is the Authorizing Official (AO). By default this is the agency head, but they can delegate this authority. In the GSA TTS, the Director of Infrastructure is the delegated AO. The AO is ultimately responsible for determining if the risk of operating the system is acceptable, and if so, issuing an Authority to Operate (ATO) for that system.
+
+The steps in the process are as follows:
+
+### Step 1: Categorize Information System
+
+The information systems' owner, working with the AO, categorizes the system based on the potential impact on the organization if the information system, or the information within it, is jeapordized. The system will end up with a category of _low_, _moderate_ or _high_, based on criteria described in [FIPS Publication 199](http://csrc.nist.gov/publications/fips/fips199/FIPS-PUB-199-final.pdf), which is summarized [here](../categorize/).
+
+### Step 2: Select Security Controls
+
+The information system's owner, working with the AO, decides which controls should be implemented. NIST's encyclopedic [Special Publication 800-53](http://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-53r4.pdf) (currently on revision 4) is the definitive guide to security and privacy controls for federal information systems. Which controls are selected is based on the following:
+
+* *The impact level of the system* (low, moderate or high). SP 800-53 provides a "baseline" set of controls for each level. The higher the level, the more controls are in scope. For systems running on cloud infrastructure, you should consult [FedRAMP's security control documentation](https://www.fedramp.gov/resources/documents-2016/).
```

Comment:
"The higher the level, the more controls are in scope" 
to
"The higher the level, the more controls and/or control enhancements are within the scope"